crates/wasm-encoder/src/core/types.rs

```diff
@@ -67,6 +67,8 @@ impl RefType {
         nullable: true,
         heap_type: HeapType::Extern,
     };
+
+    // TODO(imikushin): add all other RefTypes
```

Comment:
All of them are added now, no? Can this comment be removed?

Reply:
Silly me :)

Removed.